### PR TITLE
Add embed block test cases

### PR DESCRIPTION
**Pull request by jd-alexander**
Test cases

- Verify that inline previews are rendered properly for the top 5 most-used providers (YouTube, Twitter, Instagram, WordPress and Vimeo).

- Verify that inline previews are rendered accordingly for each alignment option.

### CODE REVIEW

**Comment by jd-alexander** [comment]
@fluiddot should I add screenshots for these tests? They seemed pretty straight forward to me, meaning the alignment options are self explanatory and for the previews, if a _breaking_ style issue is taking place it will get noticed. Let me know what you think.

File: test-cases/gutenberg/embed.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+##### TC001
+
+**Inline previews are rendered correctly for the top 5 most-used providers**
+
+#### **Precondition**
+
+Perform these tests for each of the top 5 most-used providers below. 
+ 1. YouTube
+ 2. Twitter
+ 3. Instagram
```
Comment (fluiddot) [inline test-cases/gutenberg/embed.md:14]: I'm wondering if we should warn that Instagram only works in WP.com sites or self-hosted sites connected to Jetpack, wdyt?

File: test-cases/gutenberg/embed.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+### 6. Full width
+**Steps**
+- Tap on the align options located in the toolbar.
+- Tap on "Full width".
+
+**Expected Behavior**
+- Observe that the block's width covers the entire editor and that the preview content covers the entire block.
+
+
+--------------------------------------------------------------------------------
```
Comment (fluiddot) [inline test-cases/gutenberg/embed.md]: I think we could remove the extra new lines and the line separator.

File: test-cases/gutenberg/embed.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+**NOTE:** Some embed content like Tweets won't cover the entire block because it's not designed for that purpose.
+
+### 6. Full width
+**Steps**
+- Tap on the align options located in the toolbar.
+- Tap on "Full width".
+
+**Expected Behavior**
+- Observe that the block's width covers the entire editor and that the preview content covers the entire block.
```
Comment (fluiddot) [inline test-cases/gutenberg/embed.md:169]: Similar to the "Wide width" option, some embed content won't cover the entire block so probably we could prompt the same as you added in [this line](https://github.com/wordpress-mobile/test-cases/pull/93/files#diff-9c10c7157a55a72931892f954e5318a0ffdd0bba34da15805785c37240a1f8ceR77).

**Comment by fluiddot** [comment]
> @fluiddot should I add screenshots for these tests? They seemed pretty straight forward to me, meaning the alignment options are self explanatory and for the previews, if a _breaking_ style issue is taking place it will get noticed. Let me know what you think.

I agree that the instructions are straight forward so we could omit to add the screenshots. If I had to add any I'd focus on the expected preview when changing the alignment options.

File: test-cases/gutenberg/embed.md (hunk 1 above)
Comment (jd-alexander) [reply test-cases/gutenberg/embed.md:14]: Done 7b44afe

**Comment by jd-alexander** [comment]
> > @fluiddot should I add screenshots for these tests? They seemed pretty straight forward to me, meaning the alignment options are self explanatory and for the previews, if a _breaking_ style issue is taking place it will get noticed. Let me know what you think.
> 
> I agree that the instructions are straight forward so we could omit to add the screenshots. If I had to add any I'd focus on the expected preview when changing the alignment options.

Done! I reused the screenshots from https://github.com/WordPress/gutenberg/pull/33851 since they were so nicely done there and they fit well with the test cases. Let me know if this approach is fine. Thanks!

**Comment by fluiddot** [comment]
> Done! I reused the screenshots from [WordPress/gutenberg#33851](https://github.com/WordPress/gutenberg/pull/33851) since they were so nicely done there and they fit well with the test cases. Let me know if this approach is fine. Thanks!

Great idea! Now looks perfect, thanks for adding them 🙇 .

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

I'm wondering if we should add this block to the [sanity check test suites](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-suites/gutenberg/sanity-test-suites.md), I noticed that the last blocks merged (Verse, Video, File, Audio, Reusable block) are not included so probably it's a good time to push an update, I'll open an issue as a follow-up and tackle it in a different PR.

**Comment by fluiddot** [comment]
> LGTM 🎊 !
> 
> I'm wondering if we should add this block to the [sanity check test suites](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-suites/gutenberg/sanity-test-suites.md), I noticed that the last blocks merged (Verse, Video, File, Audio, Reusable block) are not included so probably it's a good time to push an update, I'll open an issue as a follow-up and tackle it in a different PR.

I opened this https://github.com/wordpress-mobile/test-cases/pull/94 to add the missing sanity tests.

**Pull request merged**